cms/templates/course_outline.html

```diff
@@ -157,32 +157,17 @@ <h3 class="sr">${_("Page Actions")}</h3>
     <section class="content">
         <article class="content-primary" role="main">
             <div class="course-status">
-                <div class="status-release">
-                    <h2 class="status-release-label">${_("Course Start Date")}</h2>
-                    <br>
-                    <p class="status-release-value">${course_release_date}</p>
-                    <ul class="status-actions">
-                      <li class="action-item action-edit">
-                        <a href="${settings_url}" class="edit-button action-button" data-tooltip="${_("Edit Start Date")}">
-                            <span class="icon fa fa-pencil" aria-hidden="true"></span>
-                            <span class="action-button-text sr">${_("Edit Start Date")}</span>
-                        </a>
-                      </li>
-                    </ul>
-                </div>
-                <div class="status-pacing">
-                    <h2 class=status-pacing-label>${_("Course Pacing")}</h2>
-                    <br>
-                    % if context_course.self_paced:
-                        <p class="status-pacing-value">${_("Self-Paced")}</p>
-                    % else:
-                        <p class="status-pacing-value">${_("Instructor-Paced")}</p>
-                    % endif
-                </div>
-                <div class="status-highlights-enabled"></div>
                 % if should_show_checklists_page():
-                <div class="status-checklist">
+                ## set width dynamically depending upon whether course has a start date to ensure spacing look good
+                % if course_release_date == 'Set Date':
+                    <div style="width: 40%" class="status-studio-frontend">
+                % else:
+                    <div style="width: 50%" class="status-studio-frontend">
+                % endif
```

Comment:
Out of sheer curiosity - if width set to 45% all the time how does it look? Like could we split the difference and avoid the extra logic? Or is it just too big & just too small at that size?

Reply:
Just for record-keeping: we looked at the differences in widths and decided to keep it as 40%/50%.